cmake/profile.cmake

```diff
@@ -1,4 +1,30 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-include(${MBED_ROOT}/cmake/profiles/${MBED_PROFILE}.cmake)
+# Set Develop as our default type if no other selected 
```

Comment:
```suggestion
# Set the default build type if none is specified
```
I would not make the comment dependent on the select default (in case it changes).